packages/pn-landing-webapp/pages/informativa-privacy.tsx

```diff
@@ -2,13 +2,10 @@ import { useEffect } from "react";
 import type { NextPage } from "next";
 import Head from "next/head";
 
-// import { Typography, Grid, Link, Container } from "@mui/material";
-
-// const FAVICON_PATH = `${process.env.NEXT_PUBLIC_ASSETS_URL}/favicon.svg`;
-
 declare const OneTrust: any;
```

Comment:
@AndreaCimini90 can we add a type to OneTrust instead of any?

Reply:
What type?

Reply:
We can also infer the type by usage

Reply:
Done